spec/controllers/api/v1/companies_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::CompaniesController, type: :controller do
+
+  describe "GET #show" do
+    let(:response_body) { JSON.parse(response.body) }
+    let(:company) { create(:company) }
+
```

Comment:
I don't know why the project uses this syntax (which I have also never seen before), but I used it to stay consistent.